__fixtures__/example-todo-main/package.json

```diff
@@ -6,7 +6,8 @@
     "../../packages/*"
   ],
   "devDependencies": {
-    "@redwoodjs/core": "0.32.2"
+    "@redwoodjs/core": "0.32.2",
+    "inngest-setup-redwoodjs": "^0.4.1"
```

Comment:
```suggestion
    "@redwoodjs/core": "0.32.2"
```

This package shouldn't be added to Fixture. You'll need to revert yarn.lock for fixture as well.

Reply:
Also added note in readme that if your command modifies a project, be sure to remove changes in the test project.

Reply:
Done